src/utils.ts

```diff
@@ -1,2 +1,7 @@
 export const IS_WINDOWS = process.platform === 'win32';
 export const IS_LINUX = process.platform === 'linux';
+export const getPlatform = (): 'windows' | 'linux' | 'mac' => {
```

Comment:
is where any reason to have this a function instead of const? I see it is evaluated from 3 other constants and always return the same result